WARNING | 2021-12-01 22:44:49,960 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: gnutls_malloc
WARNING | 2021-12-01 22:44:49,961 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: gnutls_free
WARNING | 2021-12-01 22:44:49,973 | cle.loader | For more information about "Symbol was allocated without a known size", see https://docs.angr.io/extending-angr/environment#simdata
ERROR   | 2021-12-01 23:04:29,347 | root     | b'out of memory'
ERROR   | 2021-12-01 23:04:29,348 | root     | got an error while analyzing print_info
ERROR   | 2021-12-01 23:04:58,946 | root     | b'out of memory'
ERROR   | 2021-12-01 23:04:58,946 | root     | got an error while analyzing print_list
ERROR   | 2021-12-01 23:05:03,727 | root     | list index out of range
ERROR   | 2021-12-01 23:05:03,727 | root     | got an error while analyzing doUsageOpt
Fatal Python error: Cannot recover from stack overflow.

Current thread 0x00007f41e73d00c0 (most recent call first):
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 307 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 316 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 316 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 316 in _arg_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 335 in _ast_serialize
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 266 in _calc_hash
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/ast/base.py", line 224 in __new__
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 515 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in <listcomp>
  File "/home/ittay.al/venv/pproc/lib/python3.7/site-packages/claripy/backends/backend_z3.py", line 407 in _abstract_internal
  ...
timeout: the monitored command dumped core
Aborted
